terraform/modules/tf_stream_alert_cloudwatch/iam.tf

```diff
@@ -0,0 +1,87 @@
+# IAM Role: Allows CloudWatch Logs to put data into
+# this cluster's default Kinesis stream
+resource "aws_iam_role" "cloudwatch_subscription_role" {
+  name = "stream_alert_${var.cluster}_cloudwatch_subscription_role"
+
+  assume_role_policy = "${data.aws_iam_policy_document.cloudwatch_logs_assume_role_policy.json}"
+}
+
+// IAM Policy Document: AssumeRole for CloudWatch Logs
+data "aws_iam_policy_document" "cloudwatch_logs_assume_role_policy" {
+  statement {
+    effect  = "Allow"
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "Service"
+      identifiers = ["logs.${var.region}.amazonaws.com"]
+    }
+  }
+}
+
+// IAM Policy: Write to Kinesis
+resource "aws_iam_role_policy" "cloudwatch_kinesis_wo" {
+  name = "stream_alert_${var.cluster}_write_cloudwatch_to_kinesis"
```

Comment:
Can we make this camel case? Maybe `WriteCLWToKinesis`

Reply:
does it need to be unique? Meaning can I drop the cluster interpolation without hurting anything?

Reply:
also are you only suggesting using camel case for the role policy? or the other 'name' attributes as well?

Reply:
It just has to be unique to the role itself. And just the role_policy since `name` shows up as the descriptor in the IAM console